ldap/ldap.go

```diff
@@ -13,9 +13,9 @@ import (
 	ldap "gopkg.in/ldap.v2"
 )
 
-// userService provides authentication and authorization of users against an LDAP service. It
+// UserService provides authentication and authorization of users against an LDAP service. It
```

Comment:
I don't think this should be public.

Reply:
My rationale for making it public was to allow the Evergreen code to do a type assertion to determine if the UserManager in use is a ldap.userService and only show the buttons in that case.

Reply:
I've replaced the type assertion by saving in the UIServer struct the type of auth in use.